frontend/src/routes/Governance/policy-sets/CreatePolicySet.test.tsx

```diff
@@ -74,19 +68,66 @@ describe('Create Policy Page', () => {
       nockCreate(mockPolicySets[2]),
     ]
 
-    const placementRuleNock = [
-      nockCreate(mockPlacementRules[1], undefined, 201, { dryRun: 'All' }), // DRY RUN
-      nockCreate(mockPlacementRules[1]),
+    const mockPlacement: Placement = {
```

Comment:
Nit: I'd prefer these be kept in the `governance.sharedMocks.tsx` file to keep with existing conventions.

Reply:
So...do you want to add a Placement/PlacementBinding mock generation function to the sharedMocks file to use here? 🙂

Reply:
For the sake of time, I think I'll defer that for now. :smile: